apps/site/assets/ts/schedule/__tests__/LineDiagramTest.tsx

```diff
@@ -506,4 +507,16 @@ describe("LineDiagram", () => {
     const wrapper = renderer.create(<LineDiagram lineDiagram={lineDiagram} />);
     expect(wrapper.toJSON()).toMatchSnapshot();
   });
+
+  it("has a tooltip for a transit connection", () => {
+    const wrapper = mount(<LineDiagram lineDiagram={lineDiagram} />);
+    const stopConnections = wrapper.find(
+      ".m-schedule-line-diagram__connections a"
+    );
+    stopConnections.forEach(connectionLink => {
+      const props = connectionLink.props();
+      expect(props.title).toBeTruthy();
```

Comment:
Can we have something that asserts the title content is what we want it to be, given the data? I think we should have the `connectionName` logic tested somewhere since that's a key part of this feature.